source/NoteWindow.cpp

```diff
@@ -742,6 +756,11 @@ void NoteWindow :: MessageReceived(BMessage* message) {
 		}
 		break;
 
+		// Open a note
+		case OPEN: {
+			fOpenPanel -> Show();
+		}
+
```

Comment:
You missed a `break;`. (I don't think you intended to have a fall-through here.) Also, move this case to above SAVE_AS.